org-super-agenda.el

```diff
@@ -493,6 +494,44 @@ DATE', where DATE is a date string that
                                        ((or 'before 'on 'after) target-date))))
                    (org-super-agenda--compare-dates comparison entry-time compare-date))))))))
 
+(org-super-agenda--defgroup timestamp
+  "Group items whose bodies contain an active timestamp.
+Argument can be `t' (to match items with any timestamps),
+`past' (to match items with timestamps in the past), `today' (to
+match items with timestamps set to today), or `future' (to match
+items with timestamps in the future).  Argument may also be given
+like `before DATE' or `after DATE', where DATE is a date string
+that `org-time-string-to-absolute' can process."
+  :section-name (pcase (car args)
+                  ('t "Active timestamps")
+                  ('today "Active timestamps for today")
+                  ('future "Future active timestamps")
+                  ('past "Past active timestamps")
+                  ('before (concat "Active timestamps before " (cadr args)))
+                  ('on (concat "Active timestamps on " (cadr args)))
+                  ('after (concat "Active timestamps after " (cadr args))))
+  :let* ((now (ts-now))
+         (target-date (pcase (car args)
+                        ((or 'before 'on 'after)
+                         (make-ts :internal (org-time-string-to-absolute (cadr args))))))
+         (test (pcase-exhaustive (car args)
+                 ('t      (lambda (_) t))
+                 ('future (lambda (it) (ts>= it now)))
+                 ('past   (lambda (it) (ts<= it now)))
+                 ('before (lambda (it) (ts<= it target-date)))
+                 ('after  (lambda (it) (ts>= it target-date)))
+                 ('on     (lambda (it) (and (= (ts-year it)  (ts-year target-date))
+                                       (= (ts-month it) (ts-month target-date))
+                                       (= (ts-day it)   (ts-day target-date))))))))
```

Comment:
This is nice, but what about using `org-super-agenda--compare-dates`?

Reply:
done